### ISSUE TITLE
Open external links in new tab

### ISSUE THREAD

**Issue by ztefanie** [digitalfabrik/integreat-app#2990]
**Is your feature request related to a problem? Please describe.**
As a user i want to easily go back to integreat after opening a external link

**Describe the solution you'd like**
External links should be opened in a new link.
Make sure that best practices for opening links in a new tab are applied.

**Describe alternatives you've considered**
Keep it as it is

**Additional context**
- this is a feature request from the target group task force
- see additional information in open project: https://openproject.tuerantuer.org/notifications/details/3233/overview
- this feature request was already discussed and approved by ui/ux (both Toni and Nikola)

**Comment by bahaaTuffaha** [comment]
Thinking out loud:
For any `Link` from `react-router-dom` or <a> tag just add `target='_blank'`

**Comment by steffenkleinle** [comment]
@ztefanie we had the same discussion a few years back, see https://chat.tuerantuer.org/digitalfabrik/pl/rj7ars9mmfbg8na47eyz78z5be.
In the end, we decided to keep opening external links in the same tab. Perhaps we should check if we actually think that we should change this now. Personally I agree that it probably makes sense though.

**Comment by LeandraH** [comment]
> Thinking out loud: For any `Link` from `react-router-dom` or tag just add `target='_blank'`

@bahaaTuffaha Yup, perfect, do you want to take on this one? You'll have to figure out how to get the `target` into the `RemoteContent` :)

**Comment by bahaaTuffaha** [comment]
RemoteContent? 🤔 we either adjust the content (my guess is from cms)😇 or inject some JavaScript to webview 😈

**Comment by LeandraH** [comment]
The file is called `RemoteContent` :) We could ask the CMS team to adjust this but I think it's more of an App behavior.

**Comment by LeandraH** [comment]
By the way, external links in remote content are already marked with the class "link-external", which might be helpful here.
